content/en/blog/2022/enhancements-process-update.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: blog
+title: "Enhancements Opt-in Process Change for v1.26"
+date: 2022-09-06
+slug: enhancements-opt-in
+author: Grace Nguyen
+---
+
+# Enhancements Opt-in Process Change for v1.26 
+
+## Context and Motivations
+
+Since the inception of the Kubernetes release team, we have used a spreadsheet to keep track of enhancements for the release. The project has scaled massively in the past few years, with almost a hundred enhancements collected for the 1.24 release. This process has become error-prone and time consuming. A lot of manual work is required from the release team and the SIG leads to populate KEPs data in the sheet. We have received continuous feedback from our contributors to streamline the process.
+
+Starting at the beginning of the 1.26 release, we are replacing the sheet with an automated [GitHub project board](https://github.com/orgs/kubernetes/projects/98).
+
+## How does the Github Project Board work?
+
+The board is populated with a script gathering all KEP issues in the `kubernetes/enhancements` repo that have the tag `lead-opted-in`. The enhancements' stage and SIG information will also be automatically pulled from the KEP issue.
+
+After the KEP is populated on the Github Project Board, the Enhancements team will manually update the KEP with the label `tracked/yes`, `tracked/no` and on occasions, `tracked/out-of-tree`. The `tracked` label signify qualification to the closest milestone. For example, at the beginning of enhancements collections, `tracked/yes` means that the KEP is verified to have opted in by the Enhancements team. Closer to Enhancements Freeze, `tracked/yes` means that the KEP has satisfied all Enhancements Freeze requirements and similarly for Code Freeze, `tracked/yes` means that all code related to the KEP have been merged. The `tracked` label is reserved for the Enhancements team use only.
```

Comment:
Is there still a need for the first use case (verified to have opted in by the Enhancements team) since we now have the `lead-opted-in` label?

Reply:
For `tracked/out-of-tree`, I would omit mentioning it unless we explain how it's used.

Reply:
oof `tracked/out-of-tree` is a beast of its own kind and requires documentation else where so I'll just omit it for now. Regarding the first use case of the `tracked` label: I'm not too sure since this is our first run, the details mentioned here is the case for previous releases.

Would you like to use the `tracked` label for Enhancements Freeze qualifiers right away? I think it makes sense